infra/rush-x/src/actions/change/__test__/action.test.ts

```
@@ -0,0 +1,111 @@
+vi.mock('fs/promises');
+vi.mock('path');
+vi.mock('@coze-infra/rush-logger');
+vi.mock('../helper');
+vi.mock('../amend-commit');
+vi.mock('../../../utils/project-analyzer');
+
+describe('generateChangeFiles', () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+    // Reset environment variables
+    delete process.env.CI;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)